src/NzbDrone.Update.Test/ProgramFixture.cs

```diff
@@ -32,8 +32,9 @@ public void should_throw_if_first_arg_isnt_an_int(string arg1, string arg2)
         }
 
         [Test]
-        public void should_call_update_with_corret_path()
+        public void should_call_update_with_correct_path_windows()
```

Comment:
Is this failing? Passing on our build server at the moment, updater should be called on all OSes to make sure it launches.